src/jeta/loss.py

```diff
@@ -2,8 +2,9 @@
 import jax.numpy as jnp
 import optax
 
+
 @jax.jit
-def task_loss(model, batch):
+def task_loss(model, batch, ways):
```

Comment:
`ways` is unused?

Reply:
`ways` is [used](https://github.com/Yash-10/max/blob/22771a32c745a399b88440e062ae1942050b7014/src/jeta/loss.py#L14) below, right?